src/frontend/js/widgets/SyllabusCourseRunsList/components/SyllabusCourseRun/index.tsx

```diff
@@ -42,6 +42,12 @@ const messages = defineMessages({
 const OpenedCourseRun = ({ courseRun }: { courseRun: CourseRun }) => {
   const formatDate = useDateFormat();
   const intl = useIntl();
+  const enrollmentStart = courseRun.enrollment_start
```

Comment:
I'm worried that these changes did not break any test :thinking: 
Can you start by writing a test that fails and then fix it with this code?